docs/Welcome.md

```diff
@@ -56,6 +61,10 @@
 
 1. [Basics](./transactions/Basics.md)
 
+## US Withholding Tax
+
+1. [Basics](./us-withholding-tax/Basics.md)

```

Comment:
Does this also need a FAQ? I imagine tenants might ask questions along the lines of:

1. What if the W-8BEN draft has incorrect information?
2. What happens when the W-8BEN expires?
3. Once the party has attested the W-8BEN has the correct information, when does the W-8BEN become valid?
4. What countries do you support for W-8BENs right now?

Reply:
FAQs addd